docs-content/installing.html.md.erb

```diff
@@ -73,34 +71,74 @@ $ cf curl /v2/info
 
 If you're using a Proxy Server and SaaS controller, the two PCF apps deployed by the AppDynamics Platform Monitoring for PCF tile, AppDynamics Nozzle and the AppDynamics KPI Dashboard, require that you set environment variables to define the proxy host and port so the apps can communicate with the configured SaaS controller. 
 
-The following instructions assume the use of the Cloud Foundry CLI. You could also set the environment variables from the Apps Manager as well.
+You can configure the proxy environment variables in either in  `filename.here` or using the Apps Manager.
```

Comment:
@jeffaholmes, In what file(s) do they enter their environment variables?

Reply:
@okilker not aware of a file based option, only CF CLI (command line interface) and Apps Manager to set env var